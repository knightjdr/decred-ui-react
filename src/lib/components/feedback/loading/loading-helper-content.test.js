import LoadingHelperContent from './loading-helper-content';

describe('LoadingHelperContent', () => {
  test('Returns content', () => {
    const wrapper = LoadingHelperContent('#000', 20, 1.5);
    expect(wrapper).toMatchSnapshot();
  });
});
