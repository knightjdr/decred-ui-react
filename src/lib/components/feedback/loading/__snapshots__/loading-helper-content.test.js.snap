// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingHelperContent Returns content 1`] = `
Array [
  <svg
    className="decred-ui-loading-svg"
    style={
        Object {
            "height": 20,
            "width": 20,
          }
    }
>
    <circle
        className="decred-ui-loading-dot-left"
        style={
            Object {
                "animationDuration": "1.5s",
                "fill": "#000",
              }
        }
    />
</svg>,
  <svg
    className="decred-ui-loading-svg"
    style={
        Object {
            "height": 20,
            "width": 20,
          }
    }
>
    <circle
        className="decred-ui-loading-dot-middle"
        style={
            Object {
                "animationDelay": "0.30000000000000004s",
                "animationDuration": "1.5s",
                "fill": "#000",
              }
        }
    />
</svg>,
  <svg
    className="decred-ui-loading-svg"
    style={
        Object {
            "height": 20,
            "width": 20,
          }
    }
>
    <circle
        className="decred-ui-loading-dot-right"
        style={
            Object {
                "animationDelay": "0.6000000000000001s",
                "animationDuration": "1.5s",
                "fill": "#000",
              }
        }
    />
</svg>,
]
`;
